src/Microsoft.DotNet.Build.Tasks/PackageFiles/Build.Common.targets

```diff
@@ -8,6 +8,9 @@
 
   <PropertyGroup>
     <BuildToolsTaskDir Condition="'$(BuildToolsTaskDir)' == ''">$(ToolsDir)</BuildToolsTaskDir>
+    
+    <!-- For UAP, we need to bin place the resources as resw files to create the runner's pri file. -->
+    <ResourcesFolderPath Condition="'$(ResourcesFolderPath)' == ''">$(RuntimePath)resw</ResourcesFolderPath>
```

Comment:
No, this is only the property definition. They are being bin-placed here: https://github.com/dotnet/buildtools/pull/1601/files#diff-dccabf83e08d3281efb7ebb5798da47aR35

And that is conditoned to UAPVertical.